src/include/bigquery_sql.hpp

```diff
@@ -31,6 +31,28 @@ struct BigquerySQL {
 
 	static string BigqueryColumnToSQL(const ColumnDefinition &column);
     static string BigqueryColumnsToSQL(const ColumnList &columns, const vector<unique_ptr<Constraint>> &constraints);
+
+	static string ColumnsFromInformationSchema(const string &project_id, const string &dataset);
+};
+
+struct BigqueryPseudoColumns {
+public:
+	static const std::unordered_set<string> GetPseudoColumns() {
+		static const std::unordered_set<string> pseudoColumn = {
+			"_PARTITIONTIME",
+			"_PARTITIONDATE",
+			"_TABLE_SUFFIX",
+			"_FILE_NAME",
+			"_ROW_TIMESTAMP",
+			"_STREAMING_TIMESTAMP",
+			"_DATASTORE_ID",
+		};
+		return pseudoColumn;
+	}
+
+	static bool IsPsuedoColumn(const string &column_name) {
```

Comment:
small typo
```suggestion
	static bool IsPseudoColumn(const string &column_name) {
```

Reply:
Yea that code was already unused and outdated but I forgot to push. Thanks for the review though!